src/main/python/google_routes_db/google_routed_db_ddl.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE gr_route (
+  id BIGSERIAL PRIMARY KEY,
+  bound_northeast geography(POINT),
```

Comment:
> None of the GEOS functions support the geography type. As a workaround one can convert back and forth between geometry and geography types.
https://postgis.net/docs/using_postgis_dbmanagement.html#PostGIS_Geography

Hence, **geometry** type is better